cmd/dcos-upgrade.go

```diff
@@ -76,40 +76,40 @@ func (uc *dcosUpgradeCmd) validate(cmd *cobra.Command) error {
 
 	uc.locale, err = i18n.LoadTranslations()
 	if err != nil {
-		return fmt.Errorf("error loading translation files: %s", err.Error())
+		return errors.Wrap(err, "error loading translation files")
 	}
 
 	if len(uc.resourceGroupName) == 0 {
 		cmd.Usage()
-		return fmt.Errorf("--resource-group must be specified")
+		return errors.New("--resource-group must be specified")
```

Comment:
Just curious: for error messages that aren't synthesizing downstream `err` objects with add'l string message context, what is the value of using `github.com/pkg/errors`? I.e., what the benefit of `errors.New()` over `fmt.Errorf()`?

Reply:
`errors.New()` stores the stack information which can aid in debugging... e.g. you can `fmt.Printf("+v")` and see the full stack trace of this error.